src/Ray.jl

```diff
@@ -14,7 +14,7 @@ using Logging
 using LoggingExtras
 using Pkg
 using Serialization: Serialization, AbstractSerializer, Serializer, deserialize,
-    reset_state, serialize, serialize_type, ser_version, writeheader
+                     reset_state, serialize, serialize_type, ser_version, writeheader
 using Sockets: IPAddr, getipaddr
 
 import ray_julia_jll as ray_jll
```

Comment:
```suggestion
using ray_julia_jll: ray_julia_jll as ray_jll
```
Looks like JuliaFormatter doesn't recognize this case

Reply:
TIL